GeeksCoreLibrary/Core/Services/WiserItemsService.cs

```diff
@@ -3761,6 +3761,30 @@ public async Task<List<WiserItemFileModel>> GetItemFilesAsync(IWiserItemsService
 
             return result;
         }
+        
+        /// <inheritdoc />
+        public async Task<List<string>>GetDedicatedTablePrefixesAsync()
```

Comment:
PLease add a space before the function name.

Reply:
done